src/extras.jl

```diff
@@ -56,14 +63,16 @@ also accept them.
 * `extend::Bool=false`: when `false`, an error is raised if some values in `x` fall
   outside of the breaks; when `true`, breaks are automatically added to include all
   values in `x`, and the upper bound is included in the last interval.
-* `labels::AbstractVector=[]`: a vector of strings giving the names to use for the
-  intervals; if empty, default labels are used.
+* `labels::Union{AbstractVector,Function}=_default_formatter_`: a vector of strings giving the names to use for the
```

Comment:
I wouldn't show the name `_default_formatter_` as it's not exported. Just mentioning it in the text is OK.

Please also add line breaks at 92 chars.